comb_spec_searcher/strategies/strategy.py

```diff
@@ -563,6 +638,10 @@ def __call__(
     def can_be_equivalent() -> bool:
         return False
 
+    @staticmethod
+    def is_two_way(comb_class: CombinatorialClassType) -> bool:
+        return False
```

Comment:
Shouldn't that be true? Same for can_be_equivalent above?

Reply:
Since you have no children, it doesn't really matter

Reply:
Oh got tricked by the GitHub folding. I thought it was part of the symmetry strategy